content_scripts/link_hints.coffee

```diff
@@ -394,7 +394,7 @@ class LinkHintsMode
             clickActivator = (modifiers) -> (link) -> DomUtils.simulateClick link, modifiers
             linkActivator = @mode.linkActivator ? clickActivator @mode.clickModifiers
             # TODO: Are there any other input elements which should not receive focus?
-            if clickEl.nodeName.toLowerCase() in ["input", "select"] and clickEl.type not in ["button", "submit"]
+            if clickEl.nodeName.toLowerCase() in ["input", "select"] or clickEl.tabIndex >= 0
```

Comment:
Thanks, @gdh1995.  Two things...

1. I think we should stick with `and` here (not `or`).  I'm concerned that we could be with the focus in an `embed` (or something) and unintentionally in insert mode.  Would that happen?
2. We can lose the comment on line 396, or perhaps replace it with a comment that better describes what is going on.

Reply:
Bump @gdh1995 to resolve this conversation or respond to each point (in particular, the first comment).